packages/core/lib/services/relay-service/configs.js

```diff
@@ -0,0 +1,31 @@
+import assert from 'assert';
+import {isNumberValid} from '../../core/validation';
+
+export const validation = {
+  sendMessage({keyPairDoc, recipientDid, message}: SendMessageParams) {
+    assert(!!keyPairDoc, 'keyPairDoc is required');
+    assert(!!recipientDid, 'recipientDid is required');
+    assert(typeof message === 'string', 'invalid message');
```

Comment:
would there be value to accepting an object and serializing it? Or are you deliberately leaving serialization to the caller?